examples/conways_game_of_life/app.py

```diff
@@ -0,0 +1,85 @@
+import mesa
+
+import streamlit as st
+
+import time
+
+import pandas as pd
+
+import altair as alt
+
+import plotly.express as px
+
+import numpy as np
+from conways_game_of_life.model import ConwaysGameOfLife
+
+import pandas as pd
+
+
+import random
+
+model = st.title("Boltzman Wealth Model")
+num_ticks = st.slider(
+    "Select number of Simulation Runs", min_value=1, max_value=100, value=50
```

Comment:
Should be "Select number of simulation steps".

Reply:
this has been updated on the latest commit